### PR TITLE
Add setting to disable in-game UI + add Migraine to alt Ch2 shared enemies

### PR DESCRIPTION
**Pull request by guwahavel**
Setting is labeled "Disable In-Game UI" under the Test tab. When enabled, rendered text and door sprites associated with the BR test mod will be disabled, which is ideal for when aesthetically pleasing screenshots or footage is desired.

### CODE REVIEW

File: BasementRenovator.py [hunk 1]
```diff
@@ -6017,9 +6022,10 @@ def writeTestData(self, folder, testType, floorInfo, testRooms):
     StageType = {floorInfo.get('StageType')},
     StageName = {strFix(floorInfo.get('Name'))},
     IsModStage = {floorInfo.get('BaseGamePath') is None and 'true' or 'false'},
+    DisableUI = {settings.value("DisableInGameUI")},
```
Comment (budjmt) [inline BasementRenovator.py:6025]: can you make this write `true` or `false` explicitly? Idk what this will do when the value isn't set at all.

**Review by budjmt — CHANGES_REQUESTED** [review]
Might also be good to add a keyboard shortcut in the mod to toggle the setting, in addition to having it at launch. If you are willing to add it, please document it in [README.md under Instapreview](https://github.com/Basement-Renovator/basement-renovator?tab=readme-ov-file#how-to-use-the-interface), like [this commit](https://github.com/Basement-Renovator/basement-renovator/commit/77aeaaddb93e9b8265ff284586dd0505ac504b53)

**Comment by budjmt** [comment]
Thanks for contributing this!

File: BasementRenovator.py (hunk 1 above)
Comment (guwahavel) [reply BasementRenovator.py]: Implemented this and the keyboard toggle

**Review by budjmt — APPROVED** [review]

**Comment by budjmt** [comment]
Hmm this still seems to allow the doors to render.

**Pull request merged**